config/prod/prow/jobs/custom/branchprotector.yaml

```diff
@@ -72,8 +44,8 @@ periodics:
       command:
       - /app/prow/cmd/branchprotector/app.binary
       args:
-      - --config-path=config/branch_protector/rules.yaml
-      - --job-config-path=config/prod/prow/jobs/custom/branchprotector.yaml
+      - --config-path=../../GoogleCloudPlatform/oss-test-infra/prow/knative/config.yaml
```

Comment:
I do not enjoy that this config is in the GCP repo. Any way to roll that back out? It should be in a Knative repo.

Reply:
I don't like it in the other repo either. We have internal discussions to move the config back to the Knative repo, but haven't come to an agreement yet. I will share more details once we have a conclusion, but for now let's merge this PR first?